Dockerfile.ci

```diff
@@ -67,23 +84,23 @@ RUN cd python && bundle install
 RUN cd terraform && bundle install
 RUN cd omnibus && bundle install
 
-COPY common/ ${CODE_DIR}/common/
-COPY bundler/ ${CODE_DIR}/bundler/
-COPY cargo/ ${CODE_DIR}/cargo/
-COPY composer/ ${CODE_DIR}/composer/
-COPY dep/ ${CODE_DIR}/dep/
-COPY docker/ ${CODE_DIR}/docker/
-COPY elm/ ${CODE_DIR}/elm/
-COPY git_submodules/ ${CODE_DIR}/git_submodules/
-COPY github_actions/ ${CODE_DIR}/github_actions/
-COPY go_modules/ ${CODE_DIR}/go_modules/
-COPY gradle/ ${CODE_DIR}/gradle/
-COPY hex/ ${CODE_DIR}/hex/
-COPY maven/ ${CODE_DIR}/maven/
-COPY npm_and_yarn/ ${CODE_DIR}/npm_and_yarn/
-COPY nuget/ ${CODE_DIR}/nuget/
-COPY python/ ${CODE_DIR}/python/
-COPY terraform/ ${CODE_DIR}/terraform/
-COPY omnibus/ ${CODE_DIR}/omnibus/
+COPY --chown=${USERNAME}:${USERNAME} common/ ${CODE_DIR}/common/
+COPY --chown=${USERNAME}:${USERNAME} bundler/ ${CODE_DIR}/bundler/
+COPY --chown=${USERNAME}:${USERNAME} cargo/ ${CODE_DIR}/cargo/
+COPY --chown=${USERNAME}:${USERNAME} composer/ ${CODE_DIR}/composer/
+COPY --chown=${USERNAME}:${USERNAME} dep/ ${CODE_DIR}/dep/
+COPY --chown=${USERNAME}:${USERNAME} docker/ ${CODE_DIR}/docker/
+COPY --chown=${USERNAME}:${USERNAME} elm/ ${CODE_DIR}/elm/
+COPY --chown=${USERNAME}:${USERNAME} git_submodules/ ${CODE_DIR}/git_submodules/
+COPY --chown=${USERNAME}:${USERNAME} github_actions/ ${CODE_DIR}/github_actions/
+COPY --chown=${USERNAME}:${USERNAME} go_modules/ ${CODE_DIR}/go_modules/
+COPY --chown=${USERNAME}:${USERNAME} gradle/ ${CODE_DIR}/gradle/
+COPY --chown=${USERNAME}:${USERNAME} hex/ ${CODE_DIR}/hex/
+COPY --chown=${USERNAME}:${USERNAME} maven/ ${CODE_DIR}/maven/
+COPY --chown=${USERNAME}:${USERNAME} npm_and_yarn/ ${CODE_DIR}/npm_and_yarn/
+COPY --chown=${USERNAME}:${USERNAME} nuget/ ${CODE_DIR}/nuget/
+COPY --chown=${USERNAME}:${USERNAME} python/ ${CODE_DIR}/python/
+COPY --chown=${USERNAME}:${USERNAME} terraform/ ${CODE_DIR}/terraform/
+COPY --chown=${USERNAME}:${USERNAME} omnibus/ ${CODE_DIR}/omnibus/
```

Comment:
Seems `COPY` defaults to the file being owned by `root`, this was failing when running rspec and loading the `vcr` configuration that attempted to create a folder here: https://github.com/dependabot/dependabot-core/blob/main/common/spec/spec_helper.rb#L38 ([vcr codes](https://github.com/vcr/vcr/blob/master/lib/vcr/cassette/persisters/file_system.rb#L15))

Reply:
Yeah, the default `COPY` behaviour is surprising.

To reduce repetition we need to update this, we could just `chown -r ${CODE_DIR}` after the copy block?  I don't feel strongly about making that change.

Reply:
Hm attempted to run a `chwon -r ${CODE_DIR}` after the copy but looks like its failing and running out of disk space, reverting to the inline option https://github.com/dependabot/dependabot-core/runs/2122700298